Example/Example.xcodeproj/project.pbxproj

```diff
@@ -80,8 +74,6 @@
 		1C9404EC1B31D057001BF122 = {
 			isa = PBXGroup;
 			children = (
-				1C4B1A461B37F491008720C6 /* GaugeKit.framework */,
-				1C1E3C561B34CFE2005F495B /* GaugeKit.framework */,
```

Comment:
Thanks. I will take a look for this error also. Probably it's my mistake and it could works without links. But let's split this it in separate issue/PR.